tasks/targets.py

```diff
@@ -352,3 +352,49 @@ def update_or_create_metadata(self, _testmode=False):
                                                   column=col)
 
             session.add(coltable)
+
+
+class RepoTarget(LocalTarget):
+    def __init__(self, schema, tablename, repo_dir, resource_id, version, filename):
+        self.format = None
+        self.is_tmp = False
```

Comment:
I don't see where `is_tmp` is used :see_no_evil: Somehow related to the inheritance?